lstchain/io/event_selection.py

```diff
@@ -186,16 +183,33 @@ def apply_global_theta_cut(self, data):
         return data[data["theta"].to_value(u.deg) < self.global_theta_cut]
 
     def energy_dependent_theta_cuts(
-        self, data, energy_bins, smoothing=None,
+        self,
+        data,
+        energy_bins,
+        use_same_disp_sign=True,
+        smoothing=None,
     ):
         """
-        Evaluating an optimized energy-dependent theta cuts, in a given
-        data, with provided reco energy bins, and other parameters to
-        pass to the pyirf.cuts.calculate_percentile_cut function.
+        Evaluating energy-dependent theta cuts, in a given MC data,
+        with provided reco energy bins, and other parameters to pass to the
+        pyirf.cuts.calculate_percentile_cut function.
+
+        For MC events, the disp_sign may be reconstructed incorrectly with
+        respect to the true value, and thus resulting in a bi-modal PSF.
+        For evaluating the energy-dependent theta cuts, we want to consider,
+        only the central region of PSF. To fix this issue, by default, we apply
+        a mask on the data, so as to only use events with the same disp_sign
+        after reconstruction, for evaluating the percentile cut.
+        If the user wishes to not use this method, they can make the boolean
+        False.
```

Comment:
"Note that in that case, at low energies, where disp_sign determination is pretty uncertain, an efficiency  of 40% or larger will result in a cut which keeps the whole central part of the PSF".